common/core/desktop/src/kmx/kmx_processor.cpp

```diff
@@ -268,7 +268,7 @@ KMX_BOOL KMX_Processor::ProcessGroup(LPGROUP gp, KMX_BOOL *pOutputKeystroke)
   p = kkp->dpOutput;
   if(*p != UC_SENTINEL || *(p+1) != CODE_CONTEXT)
   {
-    for(p = decxstr((KMX_WCHAR *) u16chr(m_miniContext, 0), m_miniContext); p >= m_miniContext; p = decxstr(p, m_miniContext))
+    for(p = decxstr((KMX_WCHAR *) u16chr(m_miniContext, 0), m_miniContext); p != NULL; p = decxstr(p, m_miniContext))
```

Comment:
Confirming you don't want to check the bound
`;p != NULL && p >= m_miniContext;`

Reply:
Yes, because `decxstr` can never return a pointer before the initial pointer `m_miniContext`, unless it is `NULL`. This change has no real impact except for code clarity because `NULL < m_miniContext` of course.